api/v1beta1/watcher_types.go

```diff
@@ -65,6 +65,8 @@ type WatcherStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[0].status",description="Status"
```

Comment:
doesn't seem related, should this be split in to two PRs?

Reply:
indeed it's not, it's part of another open PR https://github.com/openstack-k8s-operators/watcher-operator/pull/68 that I rebased on top of